src/Sylius/Bundle/ApiBundle/Command/ResetPassword.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\ApiBundle\Command;
+
+class ResetPassword implements ChannelCodeAwareInterface, LocaleCodeAwareInterface
```

Comment:
```suggestion
final class ResetPassword implements ChannelCodeAwareInterface, LocaleCodeAwareInterface
```

Reply:
Commands should be left without final, as the cost of maintenance is pretty low and it allows for easier extensibility for third-party users

Reply:
```suggestion
class RequestResetPasswordToken implements ChannelCodeAwareInterface, LocaleCodeAwareInterface
```